### PR TITLE
Ranking

### PR DESCRIPTION
**Pull request by jswob**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (czikarito) [inline .vscode/launch.json]: this probably should be in gitignore?

File: app/helpers/does-it-includes.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { helper } from '@ember/component/helper';
+
+export default helper(function doesItIncludes([base, query]) {
+  return String(base).includes(String(query));
```
Comment (czikarito) [inline app/helpers/does-it-includes.js]: it should be case insensitive

File: app/templates/players/player/statistics.hbs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{{page-title "Statistics"}}
+
+<PageLayout @title={{this.model.nickname}}>
+    <h3>Games Won: {{this.model.gamesWon.length}}</h3>
+    <h3>Games Played: {{this.model.gamesPlayed.length}}</h3>
+    <h3>Win ratio: {{this.model.gamesWon.length}}/{{this.model.gamesPlayed.length}}</h3>
```
Comment (czikarito) [inline app/templates/players/player/statistics.hbs:6]: can we assume that gamesWon, gamesPlayed will be always present?

**Review by czikarito — COMMENTED** [review]
Great job! You're really fast! :D

File: app/templates/players/player/statistics.hbs (hunk 3 above)
Comment (jswob) [reply app/templates/players/player/statistics.hbs:6]: I were planning to make the entire statistics part in separate pull request. It was meant as 'dummy' content.

Comment (czikarito) [reply app/templates/players/player/statistics.hbs:6]: ah ok, no worries then

**Review by czikarito — APPROVED** [review]

**Pull request merged**